content/docs/getting_started/tutorials/dataform-101.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Dataform 101
```

Comment:
Still have this overall problem that the Dataform Web content is mixed in with the pure Dataform content. Can we move this out, or into a sub section of tutorials? (Or at least call this Dataform Web 101)

Same goes for some of the platform specific how to: version control, scheduling etc

Reply:
Hm, I'm trying to think of a way to move it while still making user flow clear for someone first visiting the site. What do you think a `Dataform 101` tutorial should contain, if not how to use the Web IDE? The only way they interact with the software is either through the Web IDE or through the CLI, and the web IDE is simpler for less technical people

Reply:
If I change the `Tutorials` sub-hearer to `Web Tutorials` is that clear enough do you think?

Reply:
How about this:

```
Getting started
  Tutorials
    Dataform Web 101
    Dataform CLI 101
    Dataform 102
```

Don't really want to rename Tutorials to Web Tutorials, because I'd like the docs to be as non Dataform Web specific as possible

Reply:
Sure